### ISSUE TITLE
cloned built & get this error:

### ISSUE THREAD

**Issue by ffischer1984** [jd-alexander/Google-Directions-Android#67]
Error:(2, 0) No service of type Factory<LoggingManagerInternal> available in ProjectScopeServices. (build.gradle)

Any ideas??

**Comment by AGKhalil** [comment]
Hey ffischer1984, were you able to figure this problem out? I know it's about 5 months ago, but if you're able to remember, that'd be great. I just cloned and downloaded the code and I have the same problem.

**Comment by ffischer1984** [comment]
yeah it was just a update for the gradle-maven-plugin - made a pull
request: https://github.com/jd-alexander/Google-Directions-Android/pull/72

AGKhalil <notifications@github.com> schrieb am So., 26. März 2017 um
21:09 Uhr:
